### DISCUSSION TITLE
Can I put ads on my vercel page?

### DISCUSSION THREAD

**Discussion by N3RDIUM** [vercel/community#2627] [Help] [answered]
### Summary

Hello there. I'm hosting my portfolio website on Vercel, with my own domain. I've got a hobby plan. My website will also include a blog. Is it okay if I add Google ads to my blog pages?

### Example

https://n3rdium.dev

### Steps to Reproduce

I'm not sure what to put here.

**Comment by swarnava** [comment, marked as answer]
Hi,

You cannot put ads on Hobby plan: https://vercel.com/docs/concepts/limits/fair-use-policy

**Reply by teju0051** [reply]
do we have to take subscription??

**Reply by swarnava** [reply]
Yes
